src/main/java/de/zalando/aruha/nakadi/domain/BatchItem.java

```diff
@@ -13,6 +12,11 @@
     public BatchItem(final JSONObject event) {
         this.response = new BatchItemResponse();
         this.event = event;
+
+        Optional.
+                of(event.optJSONObject("metadata"))
```

Comment:
Fixed 👍  https://github.com/zalando/nakadi/pull/231/commits/1b2aaead761a63afbe8edc92051fbf3f06eaa304

You were faster than CI to spot that out.